Apps/WebClient/src/ClientApp/src/components/modal/covid.vue

```diff
@@ -112,6 +112,8 @@ export default class CovidModalComponent extends Vue {
         // Prevent modal from closing
         modalEvt.preventDefault();
 
+        this.updateCovidPreference();
```

Comment:
per our team discussion today: "Dismiss covid-modal by clicking 'x' button will store the action date time in the database. It will not select any filters."